user/mods/ServerValueModifier/Presets/The Escape.json

```diff
@@ -241,19 +241,19 @@
       "PMCHMult": 1.2
     },
     "RaidMult": {
-      "MIA": 0.5,
-      "Runner": 0.8,
-      "Survived": 0.8,
-      "Killed": 0.5

```

Comment:
This is something I need to implement in PTT since this is an issue related to open-world